src/App.jsx

```diff
@@ -1,44 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState('tab-1');
+  const onTabSelected = tabs.find(tab => tab.id === activeTabId);
```

Comment:
The `onTabSelected` variable is assigned a tab object instead of a function. You should define `onTabSelected` as a function that handles the tab selection logic, and pass it to the `Tabs` component as a callback.